File: 3830-find-closest-person/find-closest-person.java
class Solution {
    public int findClosest(int x, int y, int z) {
        int d1 = x > z ? x - z : z - x;
        int d2 = y > z ? y - z : z - y;

        if(d1 < d2) return 1;
        else if(d1 > d2) return 2;
        return 0;
    }
}

File: 3830-find-closest-person/find-closest-person.cpp
class Solution {
public:
    int findClosest(int x, int y, int z) {
        int dist1 = abs(x - z);  
        int dist2 = abs(y - z);  

        if (dist1 < dist2) return 1;  
        if (dist2 < dist1) return 2;  
        return 0;                     
    }
};